doc/classes/CubemapArray.xml

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubemapArray" inherits="TextureLayered" version="4.0">
 	<brief_description>
+		A [TextureLayered] of type [code]enum TextureLayeredType.TEXTURE_LAYERED_CUBEMAP_ARRAY[/code] which uses 6, or a multiple of 6, layers.
 	</brief_description>
 	<description>
+		[TextureLayered]'s and [Cubemap]'s class references explains how a [CubemapArray] operates and how it has to be handled.
+		It is based on the class [Texture] (not [Texture2D]) and can only used as such. Its internal resource type, as well as recognized file extension, is [code]cubearr[/code].
+		The [CubemapArray] is an array of [Image]s, not cubemap levels (ie. [Cubemap]s). In Godot, technically, the [CubemapArray] is the same as the [Cubemap]. The only difference is that the [b]cubemap array has more layers[/b].
+		The array requires that you have at least 6 images (ie. layers) or a multiple amount of 6. The layers will be handled in the following order: X+, X-, Y+, Y-, Z+, Z-. See [enum VisualServer.CubeMapLayer]. Mipmaps are optional.
+		Generally, cubemap arrays provide a more efficent way for storing multiple cubemaps, than storing multiple [Cubemap]s themselves in an array.
+		In Godot, the [CubemapArray] can be used for the [Sky]. Your [ProjectSettings] value for "rendering/quality/reflections/texture_array_reflections" is reflected in [RasterizerSceneRD]s private member [code]sky_use_cubemap_array[/code]. If this member is set to [code]true[/code], the [Sky] uses the [CubemapArray]. Otherwise, the [Cubemap] logic will be used.
```

Comment:
Don't reference the internal structure of ``Sky`` here. It is not helpful for users who want to understand CubemapArrays

Reply:
Also everything below this should be cut. While it is useful information for engine dev, it is not helpful for users. A common criticism of the Godot docs is that they are too targeted at engine devs and not common people. Including a reference to every place the engine uses CubemapArrays is not useful.

Also, internally the engine does not use the Godot ``CubemapArray``, it uses the Vulkan ``CubemapArray`` (which this maps onto).

Reply:
would the aforementioned **Engine Excursion** be fine here?  
I'd include the mapping to Vulkan's `CubemapArray` you mentioned

Reply:
I simplified it, see what you think